dpnp/backend/src/queue_sycl.hpp

```diff
@@ -67,26 +151,23 @@ namespace mkl_rng = oneapi::mkl::rng;
  * initialization order is undefined. This class postpone initialization of the
  * SYCL queue and mt19937 random number generation engine.
  */
+#if 0
```

Comment:
Do we need to clean up and to remove `class backend_sycl`?

Reply:
Yes, let me push that. I thought it was done